_data/languages.yml

```diff
@@ -1,30 +1,1065 @@
---- 
+---
+-
+  code: af
+  name: Afrikaans
+  family: ine
+  scripts: [ Latn ]
+  typology:
+    - SVO, VSO
+    - analytic
+  variants: [ za ]
+
+-
+  code: ar
+  name: Arabic
+  family: sem
+  scripts: [ Arab, Syrc ]
+  typology:
+    - VSO
+    - fusional
+  variants: [ dz, bh, eg, iq, jo, kw, lb, ly, ma, om, qa, sa, sy, tn, ae, ye, sa ]
+
+-
+  code: az
+  name: Azerbaijani
+  family: trk
+  scripts: [ Latn, Cyrl ]
+  typology:
+    - SOV
+    - agglutinative
+  variants: [ az ]
+
+-
+  code: be
+  name: Belarusian
+  family: sla
+  scripts: [ Cyrl ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ by ]
+
+-
+  code: bg
+  name: Bulgarian
+  family: sla
+  scripts: [ Cyrl ]
+  typology:
+    - SVO
+    - analytic
+  variants: [ bg ]
+
+-
+  code: bn
+  name: Bengali
+  family: inc
+  scripts: Beng
+  typology:
+    - SOV
+    - inflected
+  variants: [ bd, in ]
+
+-
+  code: bs
+  name: Bosnian
+  family: sla
+  scripts: [ Cyrl, Latn ]
+  typology:
+    - SVO
+    - inflected
+  variants: [ ba ]
+
+-
+  code: ca
+  name: Catalan
+  family: roa
+  scripts: [ Latn ]
+  typology:
+    - SVO
+    - fusional
+  variants: [ es ]
```

Comment:
[ ad, es, fr ]

Andorra is the country with the highest % of Catalan speakers.

There is a .cat domain, but it's not clear if it represents a territory, and I cannot find use of "ca-cat".